### PR TITLE
Add Raven to send errors to Sentry

### PR DESCRIPTION
**Pull request by jbalboni**
Related to [1674](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1674)

Adds Raven to send errors to our Sentry url.

I don't believe the devops changes for this are live, so this should not be merged until then.

### CODE REVIEW

File: src/js/common/sentry.js
Comment (jbalboni) [inline src/js/common/sentry.js]: I think this is the logic we'd want, but need some guidance. My goal was to have this enabled only in staging and production, and not on dev, locally, or in tests.

Comment (jkassemi) [inline src/js/common/sentry.js]: This looks right in combination with the BASE_URL check. Any idea on how best to pull this out into a configuration file or set it up as a feature flag or option?

**Review by jkassemi — APPROVED** [review]

File: src/js/common/sentry.js
Comment (jkassemi) [inline src/js/common/sentry.js]: I think I'd still be in favor of keeping this active on dev.vets.gov, too. Can we just hold off on the __BUILDTYPE__ check and focus only on the BASE_URL?

**Comment by jkassemi** [comment]
Changes from devops#1589 have been deployed to all environments, so this is just held up by the configuration discussion. /cc @ayaleloehr

**Comment by ayaleloehr** [comment]
Core team said they were going to look at this today, if anyone on the West coast is still around?

cc @robbiethegeek for Monday otherwise.

**Comment by jbalboni** [comment]
@jkassemi I updated the check in sentry.js to just use the base url. Let me know if there are other changes needed for this.

**Comment by jcmeloni-usds** [comment]
@robbiethegeek-usds what does core need to do here?

**Comment by jkassemi** [comment]
Tested in dev. We'll get this out and monitor sentry for exception reports.

**Pull request merged**
